android/app/src/main/res/values/strings.xml

```diff
@@ -318,4 +318,11 @@
     <string name="status_upcoming">UPCOMING</string>
     <string name="status_ongoing">ONGOING</string>
 
+    <!--Strings with placeholders -->
+    <string name="filters_text">Filters(%d): %s</string>
+    <string name="about_fragment_license_text">%s %s</string>
```

Comment:
Just space is not a great template as it can be reused almost anywhere else. Either remove completely or rename tot general use